docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -74,3 +74,12 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
 {{% /admonition %}}
+
+
+## Permission validation on custom role creation and update
```

Comment:
Wondering if this should be an H3 under an Authorization and authentication H2?
```suggestion
## Permission validation on custom role creation and update
```

Reply:
I moved this and the refresh token part to the `Authentication and Authorization` section